go/cmd/vttestserver/cli/main.go

```diff
@@ -224,6 +224,8 @@ func New() (cmd *cobra.Command) {
 	cmd.Flags().DurationVar(&config.VtgateTabletRefreshInterval, "tablet_refresh_interval", 10*time.Second, "Interval at which vtgate refreshes tablet information from topology server.")
 
 	cmd.Flags().BoolVar(&doCreateTCPUser, "initialize-with-vt-dba-tcp", false, "If this flag is enabled, MySQL will be initialized with an additional user named vt_dba_tcp, who will have access via TCP/IP connection.")
+
+	cmd.Flags().BoolVar(&config.NoScatter, "no_scatter", false, "when set to true, the planner will fail instead of producing a plan that includes scatter queries")
```

Comment:
We are supposed to use dashes instead of underscores for new flags but this isn't really a new flag but rather we're adding it to another binary and having uniformity is better/nicer for it IMO. So I think this is fine. 🙂